scripts/docker/debian/stretch/Dockerfile

```diff
@@ -95,6 +99,15 @@ RUN echo '\
 /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/server/\n' > /etc/ld.so.conf.d/jdk.conf
 RUN ldconfig
 
+# YAEP
+RUN cd /tmp \
+    && git clone --depth 1 https://github.com/sanssecours/yaep.git \
```

Comment:
```suggestion
    && git clone --depth 1 https://github.com/ElektraInitiative/yaep.git \
```

Reply:
While I understand your reasoning, I do not think we need to add an extra copy of YAEP to the ElektraInitiative. Currently I plan to replace `sanssecours/yaep` with `vnmakarov/yaep` as soon as the author of YAEP merges https://github.com/vnmakarov/yaep/pull/16.

Reply:
Yes, original repo is even better. But please never use any private repos from repos within ElektraInitiative. We already had difficulties because of that.